code/modules/recycling/sortingmachinery.dm

```diff
@@ -38,7 +42,13 @@
 		if(EXPLODE_LIGHT)
 			SSexplosions.low_mov_atom += contents
 
-/obj/structure/big_delivery/examine(mob/user)
+/obj/item/delivery/Destroy()
+	var/turf/turf_loc = get_turf(src)
+	for(var/atom/movable/movable_content as anything in contents)
+		movable_content.forceMove(turf_loc)
```

Comment:
ideally you wouldnt do this behavior in Destroy() since its supposed to be "go away and dont leave anything behind". instead you should use another proc that moves all contents to loc and then calls qdel(src)

Reply:
@Kylerace I realized all of the edge cases I was trying to cover either call `deconstruct()` or shouldn't be possible by a user and probably should just a clean qdel anyway. Switched to using `deconstruct()`.